src/databricks/labs/ucx/install.py

```diff
@@ -714,28 +776,33 @@ def _get_collection_workspace(
         self,
         accessible_workspaces: list[Workspace],
         account_client: AccountClient,
-    ) -> Workspace | None:
+    ) -> int | None:
+        # iterate through each workspace and select workspace which has existing ucx installation
+        # allow user to select from the eligible workspace to join as collection
```

Comment:
do we need this refactoring? we may need other attributes in the `Workspace` object other than just the id?

Reply:
changed to return workspace instead of workspace_id. this method only iterates through the workspaces and allows user to select, may be there is no benefit in factoring it?